### DISCUSSION TITLE
Tidak bisa import library

### DISCUSSION THREAD

**Discussion by mochrenor** [teamalgoritma/community#216] [Q&A] [answered]
error saat run code di jupyter notebook
<img width="559" alt="Screen Shot 2022-07-21 at 3 28 57 PM" src="https://user-images.githubusercontent.com/107362857/180168580-a11d6d0c-b984-4d0d-9753-c4bb126abc0a.png">
i

**Comment by idoali** [comment]
Selamat pagi pak

Bisa dicoba install Markupsafe dengan cara `pip install markupsafe==2.0.1` di envrionment kerja bapak. Semoga dapat membantu pak

**Reply by mochrenor** [reply]
markup safe sudah terinstal, tapi masih ada eror spt ini pak
<img width="840" alt="Screen Shot 2022-07-22 at 1 07 00 PM" src="https://user-images.githubusercontent.com/107362857/180374306-1f0bc7d5-f769-44d9-a9a5-1c0cba15de2c.png">

**Comment by idoali** [comment, marked as answer]
Bisa dicoba upgrade flask terlebih dulu pak, `pip install --upgrade Flask`
